docs/sources/flow/reference/components/otelcol.processor.batch.md

```diff
@@ -0,0 +1,109 @@
+---
+aliases:
+- /docs/agent/latest/flow/reference/components/otelcol.processor.batch
+title: otelcol.processor.batch
+---
+
+# otelcol.processor.batch
+
+`otelcol.processor.batch` accepts telemetry data from other `otelcol`
+components and places them into batches. Batching improves the compression of
+data and reduces the number of outgoing network requests required to transmit
+data.
+
+> **NOTE**: `otelcol.processor.batch` is a wrapper over the upstream
+> OpenTelemetry Collector `batch` processor. Bug reports or feature requests
+> may be redirected to the upstream repository.
+
+Multiple `otelcol.processor.batch` components can be specified by giving them
+different labels.
+
+## Usage
+
+```river
+otelcol.processor.batch "LABEL" {
+  output {
+    metrics = [...]
+    logs    = [...]
+    traces  = [...]
+  }
+}
+```
+
+## Arguments
+
+`otelcol.processor.batch` supports the following arguments:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`timeout` | `duration` | How long to wait before flushing the batch. | `"200ms"` | no
+`send_batch_size` | `number` | Amount of data to buffer before flushing the batch. | `8192` | no
+`send_batch_max_size` | `number` | Upper limit of a batch size. | `0` | no
```

Comment:
Should we reuse the approach we had for the receiver component and make these `units.Base2Bytes` strings?

Reply:
I checked on this; these are actually span/log/sample counts and not size in bytes.